app/views/shared/_related_items.html+new_navigation.erb

```diff
@@ -1,3 +1,15 @@
 <div class="column-third add-title-margin">
-  <%= render partial: 'govuk_component/taxonomy_sidebar', locals: content_item.taxonomy_sidebar %>
+  <% if MainstreamContentFetcher.with_curated_sidebar.include?(content_item.base_path) %>
+    <%=
+      render(
+        partial: 'govuk_component/relted_items',
```

Comment:
@fofr thanks for the references, I've added a new commit for this: f21f4945751c987a57b71f495b6a6af9b51af236